### PR TITLE
StatsifyInc.StatsifyOverlay version 2.7.2

### PR DESCRIPTION
**Pull request by TakenMC**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16403)

### CODE REVIEW

File: manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StatsifyInc.StatsifyOverlay
```
Comment (ItzLevvie) [inline manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.installer.yaml:3]:
```suggestion
PackageIdentifier: Statsify.StatsifyOverlay
```

File: manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StatsifyInc.StatsifyOverlay
+PackageVersion: 2.7.2
+PackageLocale: en-US
+Publisher: Statsify Inc
+PackageName: Statsify Overlay
```
Comment (ItzLevvie) [inline manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.locale.en-US.yaml:7]:
```suggestion
PackageName: statsify-overlay
```
image

Other people may want to upgrade the application via the `winget upgrade` command; so the data from the Control Panel must match the one shown in the manifest.

File: manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StatsifyInc.StatsifyOverlay
```
Comment (ItzLevvie) [inline manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Statsify.StatsifyOverlay
```

File: manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: StatsifyInc.StatsifyOverlay
```
Comment (ItzLevvie) [inline manifests/s/StatsifyInc/StatsifyOverlay/2.7.2/StatsifyInc.StatsifyOverlay.yaml:3]:
```suggestion
PackageIdentifier: Statsify.StatsifyOverlay
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Comment by TakenMC** [comment]
qe

**Review by TakenMC — COMMENTED** [review]
I approve

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
My changes still need to be addressed. There should be a "Commit suggestion" button.

**Comment by ItzLevvie** [comment]
Make sure to rename the files and folders to match the `PackageIdentifier` field.

**Comment by TakenMC** [comment]
dw

**Pull request closed without merging**